roles/kubevirt/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 namespace: "kube-system"
 registry_url: "docker.io"
-docker_prefix: "{{ registry_url }}/kubevirt"
+registry_namespace: "kubevirt"
```

Comment:
I don't see `registry_namespace` variable in roles/kubevirt/README.md. 
Should it be `registry_prefix` maybe?